### ISSUE TITLE
"Unable to connect to Frigate" error when adding integration

### ISSUE THREAD

**Issue by dylanpine** [blakeblackshear/frigate#694]
Unable to add integration back after updating from 1.2 to 1.7. Issue persists in 1.8.

<img width="309" alt="" src="https://user-images.githubusercontent.com/113823/106553965-1607e480-64e0-11eb-83a8-61d9ddea683d.png">

Troubleshooting steps I've tried:

- Multiple reboots
- Check HA and Frigate logs for errors
- Verify that sidebar is working and also access on port 5000
- Disabled protection mode
- Removed Frigate and repository, rebooted host, cleared caches and reinstalled
- Confirmed MQTT is set up with a password
- Reset config to defaults and tried to add the integration

Everything works, except the integration. Everything worked before the upgrade with no issues.

**Comment by blakeblackshear** [comment]
Are you still using the beta addon? The stable version is released. You need to remove the `-beta` in the url for the stable version.

**Comment by dylanpine** [comment]
Missed the stable release, however the issue still exists after update on 1.6 stable using the non-beta url. Tried changing the port to 5001 and then adding the integration with that also. Rebooted the host multiple times.

<img width="251" alt="" src="https://user-images.githubusercontent.com/113823/106606094-5474c200-6527-11eb-9f46-95618eda9cdb.png">

Here's my HA log entry:
`21-02-02 13:14:03 INFO (SyncWorker_4) [supervisor.docker.addon] Starting Docker add-on ccab4aaf/amd64-addon-frigate with version 1.6`

And here's my Frigate log:
` * Starting nginx nginx
   ...done.
frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
peewee_migrate                 INFO    : There is nothing to migrate
frigate.app                    INFO    : Camera processor started for c1: 35
frigate.app                    INFO    : Camera processor started for c2: 36
detector.coral                 INFO    : Starting detection process: 32
frigate.edgetpu                INFO    : Attempting to load TPU as usb:0
frigate.app                    INFO    : Camera processor started for c3: 37
frigate.app                    INFO    : Capture process started for c1: 39
frigate.app                    INFO    : Capture process started for c2: 40
frigate.app                    INFO    : Capture process started for c3: 42
frigate.mqtt                   INFO    : MQTT connected
frigate.edgetpu                INFO    : TPU found`

**Comment by blakeblackshear** [comment]
I'm not sure why that dns entry wouldn't be working for you if you can access the addon with port 5000. It's definitely not 5001.

**Comment by dylanpine** [comment]
I tried changing the port to 5001 in the Frigate config just for testing purposes.

I had to make some config changes after updating from beta 1.2 to beta 1.7 to get it to start. There shouldn't be anything in the config still affecting this right??

**Comment by blakeblackshear** [comment]
It's not possible to change the port in the config any longer. Are you running an old version of the addon?

**Comment by dylanpine** [comment]
I'm on 1.6 stable. I believe this is the current version.

<img width="1118" alt="Screen Shot 2021-02-02 at 2 45 03 PM" src="https://user-images.githubusercontent.com/113823/106660257-5cebed80-6565-11eb-9766-63db6e57a5d3.png">

This is where I changed the port.

<img width="1143" alt="Screen Shot 2021-02-02 at 2 45 22 PM" src="https://user-images.githubusercontent.com/113823/106660344-7c831600-6565-11eb-8a1d-ab2c93028d74.png">

**Comment by blakeblackshear** [comment]
I _think_ you may be running an old version of the custom component. Did you update it recently?

**Comment by dylanpine** [comment]
Hmm. I removed everything Frigate including the repository under Add-on store. Then added `https://github.com/blakeblackshear/frigate-hass-addon` back as a repository and installed Frigate stable. This is version 1.6/0.8.1. Then went to configuration > integrations and added Frigate via search. I'm assuming this is the latest version of the custom component? This is where I can't connect. I even tried copying custom_components/frigate from `https://github.com/blakeblackshear/frigate-hass-integration` to my custom_components folder to be sure.

Hope I'm not missing something simple like an idiot, but I am lost. It worked so well until now!

**Comment by blakeblackshear** [comment]
I know the HomeAssistant nomenclature is super confusing, but the addon version is completely separate from the custom_component. When the WebUI was introduced, that created a breaking change for the custom_component. I would delete the `frigate` folder under `custom_components` in your config entirely and make sure you download the latest version from github again. This is most likely your issue.

**Comment by dylanpine** [comment]
That was it! Thank you for your help! Looking forward to using this new version - already loving the switches.

**Comment by HSB-collab** [comment]
I am having the same issue. I have added in the latest repository in HACS and when I try and add the integration I get the message "Unable to connect to Frigate". I am running Mosquitto as my MQTT Broker as part of Home Assistant. Could this have something to do with my setting up SSL in HA in order to access it from outside the home (using DuckDNS)? I would appreciate any help you can give.

My log says:
Logger: custom_components.frigate
Source: custom_components/frigate/config_flow.py:84
Integration: Frigate (documentation, issues)
First occurred: 6:26:11 PM (1 occurrences)
Last logged: 6:26:11 PM

Cannot connect to host frigate:5000 ssl:default [Try again]

**Comment by mwy6283** [comment]
Hi  H have you been able to solve this issue as i have been trying for days now and not having any success with replies?`21-04-05 11:52:39 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot connect to host 172.30.33.5:5000 ssl:default [Connect call failed ('172.30.33.5', 5000)]
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.check] System checks complete
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
21-04-05 11:57:53 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
21-04-05 11:59:54 INFO (SyncWorker_4) [supervisor.docker.interface] Cleaning addon_ccab4aaf_frigate application
21-04-05 11:59:54 INFO (SyncWorker_4) [supervisor.docker.interface] Removing image ccab4aaf/amd64-addon-frigate with latest and 1.13
21-04-05 11:59:54 INFO (MainThread) [supervisor.addons.addon] Removing add-on data folder /data/addons/data/ccab4aaf_frigate
21-04-05 11:59:54 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
21-04-05 11:59:54 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
21-04-05 11:59:54 INFO (MainThread) [supervisor.ingress] Update Ingress as panel for ccab4aaf_frigate
21-04-05 11:59:54 INFO (MainThread) [supervisor.addons] Add-on 'ccab4aaf_frigate' successfully removed
21-04-05 12:23:17 INFO (SyncWorker_2) [supervisor.docker.interface] Restarting homeassistant/qemux86-64-homeassistant
21-04-05 12:23:17 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API error: Received message 8:1000 is not str
21-04-05 12:23:17 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API connection is closed
21-04-05 12:23:23 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
21-04-05 12:23:26 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
21-04-05 12:23:26 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
21-04-05 12:23:53 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
21-04-05 12:23:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request initialize
21-04-05 12:23:58 INFO (MainThread) [supervisor.api.proxy] WebSocket access from a0d7b954_nodered
21-04-05 12:23:58 INFO (MainThread) [supervisor.api.proxy] Home Assistant WebSocket API request running
21-04-05 12:37:09 INFO (MainThread) [supervisor.addons] Creating Home Assistant add-on data folder /data/addons/data/ccab4aaf_frigate
21-04-05 12:37:09 INFO (SyncWorker_3) [supervisor.docker.addon] Starting build for ccab4aaf/amd64-addon-frigate:1.13
21-04-05 12:37:16 INFO (SyncWorker_3) [supervisor.docker.addon] Build ccab4aaf/amd64-addon-frigate:1.13 done
21-04-05 12:37:16 INFO (MainThread) [supervisor.addons] Add-on 'ccab4aaf_frigate' successfully installed
21-04-05 12:37:19 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
21-04-05 12:37:19 INFO (MainThread) [supervisor.ingress] Update Ingress as panel for ccab4aaf_frigate
21-04-05 12:37:23 INFO (SyncWorker_3) [supervisor.docker.addon] Starting Docker add-on ccab4aaf/amd64-addon-frigate with version 1.13
21-04-05 12:37:54 INFO (SyncWorker_0) [supervisor.docker.interface] Cleaning addon_ccab4aaf_frigate application
21-04-05 12:37:54 INFO (SyncWorker_0) [supervisor.docker.addon] Starting Docker add-on ccab4aaf/amd64-addon-frigate with version 1.13

**Comment by HSB-collab** [comment]
My problem was that I had not added the Frigate addon from the repository - see https://github.com/blakeblackshear/frigate/issues/784

**Comment by louisc32578** [comment]
I had some very bad heartburn the first time I tried frigate the integration loaded up fine.  It just gobble up more than a GIG of storage after a stormy night  So wiped it clean,  I reloaded everything not realizing I installed the Full version.  So when I went to load the integration the parameters were not the same.    
Frigate NVR | http://ccab4aaf-frigate:5000  Frigate NVR (Full Access) | http://ccab4aaf-frigate-fa:5000
My biggest hurdle is learning how to only use this as a short-term notifier.  I already own an NVR but these functions make finding parts of a video crazy fast.
